oci.go

```diff
@@ -99,3 +99,18 @@ func processRunning(pid int) (bool, error) {
 
 	return true, nil
 }
+
+func updateStoppedContainer(podStatus vc.PodStatus) error {
```

Comment:
I don't understand the naming of this function - it seems to stop the container, in which case why not just call it `stopContainer`?